photo-app/Gemfile

```diff
@@ -52,5 +52,11 @@ group :test do
   gem 'webdrivers'
 end
 
+group :production do
+  # Use Rails built-in asset pipeline
+  gem 'pg'
+  gem 'rails_12factor'
```

Comment:
Why need?

Reply:
`pg` is for heroku, `rails_12factor` was used in rails 4, so not needed, I deleted in the following [PR](
https://github.com/LeeRenJie/Learn-Ruby/pull/252)